nym-registration-client/src/builder/config.rs

```diff
@@ -207,6 +262,40 @@ fn log_mixnet_client_config(debug_config: &DebugConfig) {
         "mixnet client minimum gateway performance: {}",
         debug_config.topology.minimum_gateway_performance,
     );
+    if !debug_config.cover_traffic.disable_loop_cover_traffic_stream {
+        tracing::info!(
+            "mixnet client loop cover traffic average delay: {} ms",
+            debug_config
+                .cover_traffic
+                .loop_cover_traffic_average_delay
+                .as_millis()
+        );
+    }
+
+    tracing::info!(
+        "mixnet client average packet delay: {} ms",
+        debug_config.traffic.average_packet_delay.as_millis()
+    );
+    if !debug_config
+        .traffic
+        .disable_main_poisson_packet_distribution
+    {
+        tracing::info!(
+            "mixnet client message sending average delay: {} ms",
+            debug_config
+                .traffic
+                .message_sending_average_delay
+                .as_millis()
+        );
+    }
+    tracing::info!(
```

Comment:
That's already logged at the fn start, no need to log it again. The rest is good